optashift/image.yaml

```diff
@@ -0,0 +1,169 @@
+schema_version: 1
+
+name: "rhdm-7/rhdm70-optashift-openshift"
+description: "Red Hat Red Hat Decision Employee Rostering Business Optimizer 7.0 container image"
+version: "1.0"
```

Comment:
Please use the version 1.1 to be aligned with other images - https://github.com/jboss-container-images/rhdm-7-openshift-image/blob/rhdm70-dev/decisioncentral/image.yaml#L5